pkgs/games/dwarf-fortress/soundsense.nix

```diff
@@ -12,7 +12,7 @@
 
 stdenv.mkDerivation rec {
   version = "2016-1_196";
-  dfVersion = "0.44.10";
+  dfVersion = "0.44.11";
```

Comment:
You didn't update the hash.

Reply:
Good catch, was wondering why I needed to do that. Thought it was just assurance that all the versions are the same (which they aren't between now and the time dfhack gets updated)

Reply:
Oh nevermind it wouldn't effect the hash. I thought this was "version"

Reply:
Do you know why themes aren't installing, and is it for a similar reason?

I get "attribute 'tergel' missing" when trying to set `theme = "tergel"`.

Reply:
Oops i think that's broken. Could you try replacing this:

https://github.com/numinit/nixpkgs/blob/0bb87fdd3f7bc60102d60be9454ee62fb650a469/pkgs/games/dwarf-fortress/default.nix#L21-L24

with ```inherit themes;```?

Reply:
Yep, that fixed it, will add it to this pull. Still getting a hang of Nix...